hooks/command

```diff
@@ -29,6 +29,11 @@ readonly log_complete_timeout_sec="${BUILDKITE_PLUGIN_K8S_LOG_COMPLETE_TIMEOUT_S
 readonly use_agent_node_affinity=${BUILDKITE_PLUGIN_K8S_USE_AGENT_NODE_AFFINITY:-false}
 readonly print_resulting_job_spec=${BUILDKITE_PLUGIN_K8S_PRINT_RESULTING_JOB_SPEC:-false}
 readonly jobs_cleanup_via_plugin=${BUILDKITE_PLUGIN_K8S_JOBS_CLEANUP_VIA_PLUGIN:-}
+readonly skip_on_macos=${BUILDKITE_PLUGIN_K8S_SKIP_ON_MACOS:-}
+
+if [[ "$skip_on_macos" == "true" && $OSTYPE == darwin* ]]; then
+  exec /bin/bash -c "$BUILDKITE_COMMAND"
+fi
```

Comment:
Let's move this to the top of the file as the first thing we do? So we enforce that there really isn't any plugin logic involved.

Could we use a more generic config for this by flagging the architectures to skip? Say we have a config `BUILDKITE_PLUGIN_K8S_SKIP_ON_ARCH` that is of type array then we can regex match each item against current ostype and skip if we find a match. thinking that should allow us not to care about specifics in different use cases

Reply:
I think we could, but given your thoughts about how folks shouldn't be using this on macOS anyways maybe we should just drop the config vars all together to simplify the user side?

Reply:
That makes sense, we can check if we're on linux and if not exec yeah